migrations/001.do.sql

```diff
@@ -1 +0,0 @@
-SELECT now();
```

Comment:
this migration should be kept, as otherwise the migration library will complain that the hash of already performed migrations doesn't match with what's on disk